build.py

```diff
@@ -409,7 +409,9 @@ def pytorch_cmake_args(images):
     return [
         '-DTRITON_PYTORCH_DOCKER_IMAGE={}'.format(image),
     ]
-
+    if FLAGS.enable_gpu:
+        cargs.append('-DTRITON_PYTORCH_ENABLE_TORCHTRT=ON')
+    return cargs
```

Comment:
I just noticed there is a return statement before this. Please fix the same here. Moreover the changes from https://github.com/triton-inference-server/server/pull/3606 might supersede this.